lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -45,15 +49,18 @@ def current_shopify_session
           is_online: online_token_configured?,
         )
       rescue ShopifyAPI::Errors::CookieNotFoundError
+        ShopifyApp::Logger.info("CookiesNotFound for current Shopify session")
         nil
       rescue ShopifyAPI::Errors::InvalidJwtTokenError
+        ShopifyApp::Logger.info("Invalid JWT token for current shopify session")
```

Comment:
```suggestion
        ShopifyApp::Logger.info("Invalid JWT token for current Shopify session")
```
(same for any other places where we mention Shopify).